### PR TITLE
added Cryptome to leaks section

### PR DESCRIPTION
**Pull request by BkPankaj**
Update README.md according to mentioned issue #164

### CODE REVIEW

**Comment by BkPankaj** [comment]
@edoardottt please verify and merge the changes

**Comment by BkPankaj** [comment]
Added searchftps to code section according to mentioned issue #163 
Please check and merge the request

File: README.md [hunk 1]
```diff
@@ -161,6 +161,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [Snipplr](https://snipplr.com/all) - Code snippet search engine that allows users to search and share code snippets across various programming languages and frameworks
 - [Postman Public Collections](https://www.postman.com/explore/collections) - Explore the best APIs, collections, workspaces in the world on the Postman Public API Network
 - [ScriptMafia](https://scriptmafia.org) - Download full nulled scripts
+- [Searchftps](https://www.searchftps.net/) - Online tool for searching and accessing files on FTP servers
```
Comment (edoardottt) [inline README.md:164]: Delete this, already handled by https://github.com/edoardottt/awesome-hacker-search-engines/pull/166

File: README.md [hunk 2]
```diff
@@ -307,6 +308,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [Exposed Forum](https://exposed.vc/) - The premier Databreach discussion & leaks forum
 - [Distributed Denial of Secrets](https://ddosecrets.com/) - Journalist 501(c)(3) non-profit devoted to publishing and archiving leaks
 - [Have I Been Zuckered](https://haveibeenzuckered.com/) - Facebook Data Breach Checker
+- [Cryptome](https://cryptome.org/) - Document leaks, government transparency, surveillance information source
```
Comment (edoardottt) [inline README.md:311]: The description should be `Documents for publication that are prohibited by governments worldwide`

**Review by edoardottt — CHANGES_REQUESTED** [review]

**Pull request closed without merging**